.openapidoc/package.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I always use `husky` and `lint-staged` to fire off pre-commit lint checks. I'd recommend it even more strongly in this case as this is code that isn't going to be touched very often, so the temptation to write bad/unmaintainable code will be higher than code that people are reading all the time. 

See this as an example for how I set it up for `circle-build-monitor`. That's a TypeScript project, but all the tools I'm using stay the same for JS, just the eslint config would need to change a little... https://github.com/PegaSysEng/circle-build-monitor/blob/master/package.json#L40-L50